corehq/apps/locations/forms.py

```diff
@@ -725,3 +726,19 @@ def save(self):
                 if name.startswith('relation_distance_')
             }
         )
+
+
+def to_list(value):
+    """
+    Returns ``value`` as a list if it is iterable and not a string,
+    otherwise returns ``value`` in a list.
+
+    >>> to_list(('foo', 'bar', 'baz')) == ['foo', 'bar', 'baz']
+    True
```

Comment:
A clumsy way to write a doctest, but this allows it to pass in Py2 and Py3 ... for the next 30 days.